assets/messages.yml

```diff
@@ -318,9 +316,26 @@ messages:
         However, this issue is {color:#FF5722}*Invalid*{color}.
 
         This is an account issue. We do not have the tools to help you with this issue on this bug tracker.
-        Please have a look at the [Community Standards|https://www.minecraft.net/en-us/community-standards] for guidelines that players are expected to follow.
+        Please have a look at the [Community Standards|https://www.minecraft.net/community-standards] for guidelines that players are expected to follow.
         To learn more about player reporting, potential causes for a ban, and instructions regarding how to request a case review of a ban, please visit the [Minecraft: Java Edition Player Reporting FAQ page|https://aka.ms/chatreportingfaq].
 
+        %quick_links%
+      fillname: []
+  cannot-reproduce:
+    - project:
+      - bds
+      - mc
+      - mcl
+      - mcpe
+      - realms
+      - web
+      name: Cannot Reproduce
+      shortcut: cr
+      category: misc
+      message: |-
+        *Thank you for your report!*
+        ~Unfortunately, the described issue cannot be reproduced. Therefore, this ticket is being resolved as {color:#d04437}*Cannot Reproduce*{color}.~
+
```

Comment:
Add something saying

"If you can reproduce this issue in the latest release and/or development snapshot please make a new bug report containing steps to reproduce"

Or something like that. Like how the incomplete mesasge does it